bedrock/settings/base.py

```diff
@@ -126,8 +227,7 @@ def JINJA_CONFIG():
     path('media'),
 )
 
-JINGO_EXCLUDE_APPS = (
-    'admin', 'registration', 'rest_framework', 'rna', 'waffle')
+JINGO_EXCLUDE_APPS = ('admin', 'registration', 'rest_framework', 'rna', 'waffle')
```

Comment:
It's a tiny nit you could feel free to ignore, and I'm not demanding you change it, but a vertical list would be nice, thanks. It may not quite evoke @willkg's waterfall, but perhaps this will:
```


                                                            ---\=,</
..,.,.,,.,.,.,,.,._,,,____,,,...,.,.,,.....,....,.,..,,.  ,-=--'\\_\/
""""""""""############z_ _`"""########################' ,---'>__,>,_`-.     |
       :  |  `"""V#######,,_ `-  ""##################' --z--;" /_/  `. `.   |
          |          `/"""".`|`|| } }|.""""""""|"""""  --'//`/'  `    \  '. |
    :          :      |:     ||   |  |  :   :  |   :   ,_\---_\._   :  `.\ |/
                   :  /  :   |  |   || :  :      :    //--'> ___ ``-,_   \  \
         `"^            :  ` ||   || |   :  :         '=-`',' / `-, __`-. |
      :    :          : :  : |  |    ||    :     ---  //7;<\     / ,--._ ` |
         .,      :        :  |   ||| || '       :     -/;\'/` -='/|(    \ \
        %#'            :    `| |||    |  :   :      :  // '\   // | `    | |
    :         :   `'   :  :  || # | |||    :            `    .        :  | 
                         :   | ||#|#| | ':    :    :             :       ` 
         '#"      :   :    : | ||,|, ||   :      /        :           |:  ||
  ""'                    :  \\|\ X XX///`      :|   :    |   :      : |   |
        :             :  / >\\> <\/\< </</==::_ |' ,`  ,.|_____|______|`--|
   :         :   |    ,''{` /"^"/'/\^\'"\ --  """==;:zz,,_;__ ,; : , ,,hjm|
        |'       |`--', ',`--,,_    -_ --.  `--.     __ ""`"=;;--=--=;;-==||
        |    ,,--' ''  ,` ,',' ,'`--,._    -=-  /%%\.___     ---          |
  :     |---'  '  ' ,' ,`` , .'`.,,` , ';'--,._   `-=='  ~~  __ __ _,  ~~ |
     :,-',' '` `` ,  .`, ,',' ', ` , ,','',`, `'"`--,,_____..__          / |
,,---',  ' ' , , '  ', , ` ' ,' `, ,  , ` '` ',`,`,` ,',' ',, ,`""'--~',' /
 
```